src/components/Accordion.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import { useState } from "react";
+import { ChevronDownIcon, ChevronLeftIcon } from "@heroicons/react/24/outline";
+
+interface AccordionItem {
```

Comment:
Don't have sure if this is the best way of using TS in this case.

Reply:
Oh, this is the same type I commented on above.

When using shared types, we usually move them to our `shared` folder of `entities` folder, which holds the Domain Entities of our application. 

Domain Entities are objects that are internal to our app, like Users, Tasks, Meetings, and so on.

Also, I think Accordion is not a good name, as the type is actually a description of a Task, and accordion is only a UI element.